<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../sample.ico</normaloff>../sample.ico</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.323864 rgba(85, 85, 255, 255), stop:1 rgba(170, 170, 255, 255));
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color: white;</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Sign in or create an account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_login {
border-radius: 15px;
background-color: rgb(208, 208, 208);
font: 10pt &quot;MS Shell Dlg 2&quot;; 
}

QPushButton:hover#btn_login {
background-color:  rgb(0, 170, 255);
color: white; 
}</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create_account">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_create_account {
border-radius: 15px;
background-color: rgb(208, 208, 208);
font: 10pt &quot;MS Shell Dlg 2&quot;; 
}

QPushButton:hover#btn_create_account {
background-color:  rgb(0, 170, 127);
color: white; 
}</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>229</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
